Library/Formula/autojump.rb

```diff
@@ -1,42 +1,40 @@
 class Autojump < Formula
   homepage "https://github.com/joelthelion/autojump"
-  url "https://github.com/joelthelion/autojump/archive/release-v22.2.2.tar.gz"
-  sha1 "d23d482077049fb07dcdc1e7764694f95937db24"
+  url "https://github.com/joelthelion/autojump/archive/release-v22.2.4.tar.gz"
+  sha1 "df9ff56e128efb8a8e1af574dbac9e4b3c47c1d6"
 
   head "https://github.com/joelthelion/autojump.git"
 
   def install
-    inreplace "bin/autojump.sh", " /usr/local/share/autojump/", " #{prefix}/etc/"
+    system "./install.py", "-d", prefix, "-z", zsh_completion
 
-    libexec.install "bin/autojump"
-    libexec.install "bin/autojump_argparse.py", "bin/autojump_data.py", "bin/autojump_utils.py"
-    man1.install "docs/autojump.1"
-    (prefix/"etc").install "bin/autojump.sh", "bin/autojump.bash", "bin/autojump.zsh",
-                           "bin/autojump.fish", "bin/autojump.tcsh"
-    zsh_completion.install "bin/_j"
+    # Backwards compatibility for users that have the old path in .bash_profile
+    # or .zshrc
+    (prefix/"etc").install_symlink prefix/"etc/profile.d/autojump.sh"
 
-    bin.write_exec_script libexec+"autojump"
+    libexec.install bin
+    bin.write_exec_script libexec/"bin/autojump"
   end
 
   def caveats; <<-EOS.undent
     Add the following line to your ~/.bash_profile or ~/.zshrc file (and remember
     to source the file to update your current session):
-      [[ -s $(brew --prefix)/etc/autojump.sh ]] && . $(brew --prefix)/etc/autojump.sh
```

Comment:
I don't like such change. As in fact it breaks backwards compatibility. Users upgrade their `autojump`, then suddenly it's not working until they realize they should change their shell configuration.